### DISCUSSION TITLE
Is it possible to set the Performance option step in config?

### DISCUSSION THREAD

**Discussion by aidec** [lllyasviel/Fooocus#3308] [Q&A] [answered]
I am currently modifying Steps through modules/flags. Is it possible to set this in the Config file?

<img width="404" alt="image" src="https://github.com/user-attachments/assets/899e584b-6d89-4989-930d-4853fafac649">

**Comment by mashb1t** [comment, marked as answer]
Yes, you can override the steps in the config (works) or a preset (even better) using the key `default_overwrite_step`. Be aware that this overrides the steps for all performances, which may not be what you want.

**Reply by aidec** [reply]
Thanks for your reply, I'll try it
can't each setting be different?
like
QUALITY :40
SPEED:25
.....

**Reply by mashb1t** [reply]
Feel free to adjust the code to your liking. As mentioned, you can only override default_overwrite_step on a global level, so it's advised to create multiple presets for different performance step overrides or manually set steps override in the developer debug mode (same purpose).

**Reply by aidec** [reply]
> 請隨意根據您的喜好調整程式碼。如前所述，您只能在全域層級覆寫default_overwrite_step，因此建議為不同的效能步驟覆寫建立多個預設，或在開發人員偵錯模式下手動設定步驟覆寫（相同目的）。

Got it, thank you very much
